articles/custom-domains/set-up-cloudfront.md

```diff
@@ -0,0 +1,57 @@
+---
+toc: true
+name: Configure AWS CloudFront for Use as Reverse Proxy
+description: How to set up AWS CloudFront for use as the custom domain proxy for Auth0
+---
+# Configure AWS CloudFront for Use as Reverse Proxy
+In this article, we will show you how to configure AWS CloudFront for use as the reverse proxy with custom domain names for your Auth0 tenant.
+
+Log in to AWS, and navigate to [CloudFront](https://console.aws.amazon.com/cloudfront).
+
+![](/media/articles/custom-domains/aws/cloudfront.png)
+
+Click **Create Distribution**.
+
+You can choose the delivery method for your content. Click **Get Started** under the **Web** section.
+
+![](/media/articles/custom-domains/aws/delivery-method.png)
+
+Next, configure your distribution settings. 
+
+Under **Origin Settings**, here are the values you'll need to change:
+
+| Parameter | Value |
+| - | - |
+| Origin Domain Name | Set to `custom-domain-id.edge.tenants.auth0-region.auth0.com`. Be sure to replace the `custom-domain-id` and `auth0-region` placeholder values |
+| Origin ID | A description for the origin. This value lets you distinguish between multiple origins in the same distribution and therefore must be unique. |
+| Origin Protocol Policy | Set to **HTTPS Only** |
+
```

Comment:
Is it possible to format this similar to how you have it in Azure CDN? 
https://github.com/auth0/docs/pull/5216/files#diff-12ece6d103347e7ccba7222486739c34R44

With the warning box about non-US regions and omit the auth0-region placeholder?